crates/bevy_ui/src/ui_node.rs

```diff
@@ -15,8 +15,9 @@ use std::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Sub, SubAssign};
 #[derive(Component, Debug, Clone, Default, Reflect)]
 #[reflect(Component, Default)]
 pub struct Node {
-    /// The size of the node as width and height in pixels
-    pub size: Vec2,
+    /// The size of the node as width and height in pixels <br>
+    /// This Field should only read by the user
+    pub calculated_size: Vec2,
```

Comment:
If the field should only be read by the user, perhaps we should consider removing the `pub` or making it `pub(crate)` for example and providing an accessor? Anyone have thoughts on that?

Reply:
I also think that, looking in the code i didn't see anywhere outside of bevy_ui module that changes `calculated_size`.
I think is a good idea to remove `pub` and create a new method in Node for read-only purpose.